pom.xml

```diff
@@ -47,10 +47,13 @@
 
   <properties>
     <!-- alphabetized properties for versions -->
+    <version.guava>15.0</version.guava>
```

Comment:
This is intended for open sourcing so I think we should commit to that and assume it will be used and remove the Guava dep, especially when it is for something minor.  If we needed it for, say the Guava cache, then that would be a DGR.